pygdf/datetime.py

```diff
@@ -39,6 +40,19 @@ def __init__(self, data, mask=None, null_count=None, dtype=None):
         self._inverse_precision = 1e3
         self._pandas_conversion_factor = 1e9 * self._precision
 
+    def serialize(self, serialize):
```

Comment:
It is a little odd to have an argument with the same name as the function.

Reply:
It's existing pattern in the code base.  We can fix it in a separate PR since this PR is rather urgent.   I'm open to suggestion for new name.   Same goes for the `deserialize` classmethod.

Reply:
There are some new API in `distributed` like the `register_generic` that may make all this simpler.  See http://distributed.dask.org/en/latest/serialization.html?highlight=register_serialization

Reply:
Seems reasonable to me.